.NET/Microsoft.Recognizers.Definitions.Common/Dutch/DateTimeDefinitions.cs

```diff
@@ -239,7 +239,7 @@ public static class DateTimeDefinitions
       public const string EachPrefixRegex = @"\b(?<each>(iedere|elke|eenmaal per)\s*$)";
       public static readonly string SetEachRegex = $@"\b(?<each>(iedere|elke|om\s+de)\s*(?<other>\s+andere)?\s*(week\s*(?={WeekDayRegex}))?)";
       public const string SetLastRegex = @"(?<last>volgende?|komende|aankomende|aanstaande|deze|huidige|aanstaande|vorige?|verleden|laatste)";
-      public const string EachDayRegex = @"^\s*(iedere|elke)\s*dag\b";

```

Comment:
Why remove all these string start markers?

Reply:
To have the flexibility to recognize "every day at 5pm" as set type even if "every day" is not at starting of sentence.